### ISSUE TITLE
Angular is deprecated - please migrate to React

### ISSUE THREAD

**Issue by tolzhabayev** [michaeldmoore/michaeldmoore-multistat-panel#71]
# Hello :wave: from Grafana Labs

We have detected that your plugin might still be using `AngularJS` and recommend migrating it to `React`, so that it may continue to operate successfully in future versions of Grafana.

You can read more on why and how by visiting our documentation: https://grafana.com/docs/grafana/latest/developers/angular_deprecation/

You can reach out to us with any questions via slack: [slack.grafana.com](https://slack.grafana.com) - channel `#plugins` or our [plugin development community forum](https://community.grafana.com/c/plugin-development/30)

**Comment by michaeldmoore** [comment]
Yes, thanks for the update.  I've been looking at this and working on a
react-based version, but it all seems like too much work right now, so I
expect to retire the plugin once Angular support is discontinued.

In particular, I find the configuration system under the React framework to
be far too difficult to manage for any panel with more than a few
configuration options.  My panel has hundreds of configurable options, so
this would require an awful lot of work.
